test/kitchen/drivers/ec2-driver.yml

```diff
@@ -15,20 +15,20 @@ provisioner:
 driver:
   name: ec2
   aws_ssh_key_id: <%= ENV['KITCHEN_EC2_SSH_ID'] %>
-  security_group_ids: <%= [ENV['KITCHEN_EC2_SG_IDS']] || ["sg-7fedd80a","sg-46506837"] %>
+  security_group_ids: <%= ENV['KITCHEN_EC2_SG_IDS'] || ["sg-7fedd80a","sg-46506837"] %>
   region: <%= ENV['KITCHEN_EC2_REGION'] ||= "us-east-1" %>
   instance_type: <%= ENV['KITCHEN_EC2_INSTANCE_TYPE'] ||= 't3.xlarge' %>
   associate_public_ip: false
   subnet_id: <%= ENV['KITCHEN_EC2_SUBNET'] ||= 'subnet-b89e00e2' %>
-  iam_profile_name: <%= ENV['KITCHEN_EC2_IAM_PROFILE_NAME'] ||= nil %>
+  iam_profile_name: <%= ENV['KITCHEN_EC2_IAM_PROFILE_NAME'] ||= '' %>
```

Comment:
`nil` is the default value according to the driver docs. Why change to `''`?

Reply:
Because when I was running it w/o a profile name, that didn't work (created a parse error), and replacing it with `''` did.